.storybook/optionConfig.js

```diff
@@ -1,20 +1,31 @@
-import { addDecorator } from '@storybook/react';
-import { withOptions } from '@storybook/addon-options';
+import { addParameters, addDecorator } from '@storybook/react';
+import Theme from './theme';
+import { withA11y } from '@storybook/addon-a11y';
+import { addReadme } from 'storybook-readme';
+import centered from '@storybook/addon-centered/react';
+import { configureReadme } from 'storybook-readme';
+import { css } from '@emotion/core';
 
-addDecorator(
-  withOptions({
-    name: 'Radiance UI',
-    url: '#',
-    goFullScreen: false,
-    showStoriesPanel: true,
-    showAddonPanel: true,
-    showSearchBox: false,
-    addonPanelInRight: true,
-    sortStoriesByKind: false,
+addDecorator(withA11y);
+addDecorator(addReadme);
+configureReadme({ DocPreview: ({ children }) => <div css={css`padding: 1rem;`}>{children}</div>, StoryPreview: ({ children }) => <div css={css`padding: 1rem;`}>{children}</div>})
```

Comment:
v4 appeared to have 8px of padding added to the READMEs. v5 got rid of that, so this adds a bit of breathing room.